content/rs/installing-upgrading/_index.md

```diff
@@ -119,6 +120,70 @@ or [join]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}})
 
     For geo-distributed Active-Active replication, create an [Active-Active]({{< relref "/rs/administering/database-operations/create-crdb.md" >}}) database.
 
+### Custom installation directories
+
+During the installation you can specify the directories for the RS files to be installed in.
+The files are installed in the `redislabs` directory in the path that you specify.
+
+{{% note %}}
+- Custom installation directories are supported on RedHat Enterprise Linux versions 6 and 7.
+- When you install with custom directories, the installation does not run as an RPM file.
+- If a `redislabs` directory already exists in the path that you specify, the installation fails.
+- All nodes in a cluster must be installed with the same file locations.
+{{% /note %}}
+
+You can specify any or all of these file locations:
+
+| Files               | Installer flag | Example parameter | Example file location |
+| ------------------- | -------------- | ----------------- | --------------------- |
+| Binaries files      | --install-dir  | /opt              | /opt/redislabs        |
+| Configuration files | --config-dir   | /etc/opt          | /etc/opt/redislabs    |
+| Data and log files  | --var-dir      | /var/opt          | /var/opt/redislabs    |
+
+{{% expand "These files are not in the custom directories:" %}}
```

Comment:
Paz wanted to mention why (since they are OS directories). And maybe more input from Gidon on it